### PR TITLE
[4.0] Improve list view menus

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #22093 - Another possible solution
It replaces my  former PR https://github.com/joomla/joomla-cms/pull/20787 which can be closed.

### Summary of Changes
The link to the respective menuitems is moved to an own column.

### Testing Instructions
Compare menu overview before and after patch. Make sure that links go to the correct menu items and that ACL permissions are respected

### Actual vs.  Expected result
menu-before-after

### Documentation Changes Required
yes

### CODE REVIEW

File: administrator/components/com_menus/tmpl/menus/default.php [hunk 1]
```diff
@@ -60,23 +60,26 @@
 								<th scope="col">
 									<?php echo HTMLHelper::_('searchtools.sort', 'JGLOBAL_TITLE', 'a.title', $listDirn, $listOrder); ?>
 								</th>
-								<th scope="col" style="width:10%" class="nowrap text-center">
+								<th scope="col" style="width:5%" class="text-center">
+									<?php echo Text::_('COM_MENUS_MENUS'); ?>
+								</th>
+								<th scope="col" style="width:10%" class=text-center">
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/menus/default.php]: Add `"` before `text-center`

File: administrator/components/com_menus/tmpl/menus/default.php [hunk 2]
```diff
@@ -90,23 +93,25 @@
...
-											<?php echo $this->escape($item->menutype); ?>
+											<?php echo $this->escape($item->title); ?>
 										<?php endif; ?>
 									</div>
-								</th>
+								</td>
+								<td class="text-center btns">
+									<?php if ($canManageItems) : ?>
+										<a href="<?php echo Route::_('index.php?option=com_menus&view=items&menutype=' . $item->menutype); ?>">
+											<span class="fa fa-list text-center" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('COM_MENUS_MENUS'); ?></span>
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/menus/default.php]: Move 2nd `<span>` to the next line. Remove `text-center` since it is already in `<td>`.

**Comment by brianteeman** [comment]
can you updqte your branch please. there shouldnt be 200 files changed for this pr.

**Comment by brianteeman** [comment]
i really like this improvement. for me it solves the usability issue and i look forward to not clicking on the wrong link.

**Comment by chmst** [comment]
Maybe someone can update this branch for me? I cannot do it in my repo.

**Comment by brianteeman** [comment]
Ok let me try

**Comment by brianteeman** [comment]
My attempt to update the branch directly here did not resolve the problem. It probably needs a --force which can't be done in the ui here. For ease I would suggest creating a new pr on the current branch and applying your changes there - iirc it was just two files.

**Pull request closed without merging**